test/functional/cmdLineTests/defaultLazySymbolResolution/playlist.xml

```diff
@@ -45,9 +45,7 @@
 			<type>native</type>
 		</types>
 		<subsets>
-			<!-- disable for 15+ https://github.com/eclipse/openj9/issues/11076 -->
-			<subset>8</subset>
-			<subset>11</subset>
+			<subset>8+</subset>
 		</subsets>
```

Comment:
By default, all versions are included, and `subsets` section can be removed completely.

Reply:
Updated